packages/devtools/lib/src/tables.dart

```diff
@@ -659,14 +660,52 @@ class TreeTable<T extends TreeNode<T>> extends Table<T> {
         }
         insertIndex++;
       }
-      node.isExpanded = true;
+      node.expand();
     }
 
     expand(dataObject);
 
     _selectedObject ??= dataObject;
     setRows(data);
   }
+
+  void expandAll() {
+    // Make a copy so that we are not concurrently modifying the tree data.
```

Comment:
I don't understand the comment - this makes a new variable referencing the same list instance, not a copy.